ports/mimxrt/boards/MIMXRT1010_EVK/MIMXRT1010_EVK_af.csv

```diff
@@ -0,0 +1,22 @@
+Pad,ALT0, ALT1, ALT2, ALT3, ALT4, ALT5, ALT6, ALT7, ALT8, ALT9,ADC,ACMP,Default
```

Comment:
I think this file should go up one directory, in `ports/mimxrt/boards/`, because it's useful for all RT1010-based boards, not just this EVK.

Reply:
I do not think this is a good idea, frankly because the content of these files is specific to each board. There are overlaps but that's it. I actually only added the pins that are available through the Arduino Header (for the *_EVK boards). Therefore it is not useful in my perspective as a "generic" documents for a chip variant. At least not for now.